config/owasp/suppressions.xml

```diff
@@ -55,4 +55,14 @@
         </notes>
         <cve>CVE-2022-33915</cve>
     </suppress>
+    <suppress until="2030-01-01">
```

Comment:
I think there are others suppresions needed to pass the jenkins build, also snakeyaml related